src/util/api.js

```diff
@@ -65,6 +57,13 @@ const post = (path, body) => {
         return res.json();
       }
       return res.text();
+    })
+    .then(res => {
```

Comment:
If I understand this correctly, this is not anymore "Response" object but resolved data (either res.text() or res.json()).

I'd suggest that you change the param name to `data`.